cypress/integration/EnrollmentAddEventPage/EnrollmentAddEventPageNavigation.feature

```diff
@@ -1,5 +1,6 @@
 Feature: User interacts with Enrollment Add event page
   Scenario: The user can land on the enrollment add event page.
-    Given you land on the enrollment event page by having typed #/enrollmentEventNew?programId=IpHINAT79UW&orgUnitId=DiszpKrYNg&teiId=EaOyKGOIGRp&enrollmentId=IMEGtAd4DOj&stageId=ZzYYXq4fJie&eventId=vDGrlVeyE2Q
+    Given you land on the enrollment add event page by having typed /#/enrollmentEventNew?programId=IpHINAT79UW&orgUnitId=DiszpKrYNg8&teiId=tIJu6iqQxNV&enrollmentId=CCBLMntFuzb&stageId=A03MvHHogjR&eventId=V1CerIi3sdL
```

Comment:
Hey, 

I just noticed you have the `eventId` in the URL. When we create a new event, we don't know the `eventId` yet therefore it should not be part of the page URLs parameters. You can check the correct pattern in the [design docs](https://docs.google.com/document/d/1I9-xc1oA95cWb64MHmIJXTHXQnxzi1SJ3RUmiuzSh78/edit#heading=h.r4doz564mcx5). I am sorry I didn't notice this in your previous PR https://github.com/dhis2/capture-app/pull/1880 😅
Thank you!

Reply:
Hey, 
Everything looks good now! I just have one last comment. I think that `eventId` should also be removed from the Cypress scenario URL. 
Thank you for your patience with all my suggestions! 😁